services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx

```diff
@@ -63,6 +69,7 @@ export class Tooltip extends React.Component<TooltipProps, {}> {
   render() {
     const { tooltipTriggerText, tooltipTriggerTextClass, tooltipTriggerStyle, tooltipTriggerTextStyle, isTabbable } = this.props
     const tabIndex = isTabbable ? 0 : null;
+    const isOnMobile = window.innerWidth < 770;
```

Comment:
Is this going to break if someone's desktop browser is < 770px?

Regardless of the answer, `770` should be a constant, not a magic number, and ideally referenced from a global file (e.g. constants.ts)

Looks approvable otherwise.